src/Microsoft.DotNet.Interactive.AIUtilities/TextChunkingExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using Microsoft.DeepDev;
+
+namespace Microsoft.DotNet.Interactive.AIUtilities;
+
+public static class TextChunkingExtensions
```

Comment:
Thoughts on a better name? What else is going to end up in the category of string operations?

Reply:
mm probably will be template parameter parsing?